es-app/src/SystemData.cpp

```diff
@@ -60,7 +60,7 @@ void SystemData::setIsGameSystemStatus()
 	// we exclude non-game systems from specific operations (i.e. the "RetroPie" system, at least)
 	// if/when there are more in the future, maybe this can be a more complex method, with a proper list
 	// but for now a simple string comparison is more performant
-	mIsGameSystem = (mName != "retropie");
+	mIsGameSystem = (mName != "retropie") && (mName != "kodi");
```

Comment:
The random feature could be technically used by people who set up a Kodi system (eg to play a random video from a list there).